src/app/services/ix-slide-in.service.ts

```diff
@@ -32,9 +40,13 @@ export class IxSlideInService {
     this.slideInComponent = modal;
   }
 
-  open<T>(modal: Type<T>, params?: { wide: boolean }): T {
+  open<T>(modal: Type<T>, params?: { wide: boolean; data?: { [key: string]: unknown } }): SlideInServiceRef<T> {
     this.modalType = modal;
-    return this.slideInComponent.openSlideIn(modal, params);
+    return {
+      componentInstance: this.slideInComponent.openSlideIn(modal, params),
+      close: this.close,
+      afterClosed$: this.onClose$,
+    };
```

Comment:
The idea is to be able to tell when a specific slide in has been closed.
For example, in material dialogs I can open individual dialog, tell when it was closed and receive data from it.
```
this.dialog.open(MyDialog)
  .afterClosed()
  .subscribe((result) => {
    console.log('My Dialog was closed with result', result);
  });
```

This is not possible right now. `onClose$` will emit for every slide in closed, not just the one I'm opening.